Test/Invoke-WhiskeyPipeline.Tests.ps1

```diff
@@ -297,6 +297,20 @@ BuildTasks:
     }
 }
 
+Describe 'Invoke-WhiskeyPipeline.when task has a default property' {

```

Comment:
Add some tests with some real-world command line examples, e.g.

```
robocopy "C:\Some File" 'D:\Some Other\File' /MIR
```

We want tests to ensure the YAML is getting parsed correctly.

Reply:
I've added a couple tests to ensure the property gets passed as defined.